src/substrait/textplan/parser/SubstraitPlanRelationVisitor.cpp

```diff
@@ -316,6 +316,13 @@ bool isRelationEmitDetail(SubstraitPlanParser::Relation_detailContext* ctx) {
       nullptr;
 }
 
+bool isAggregate(const SymbolInfo* symbol) {
```

Comment:
Why are there two implementations of this function (PlanPrinterVisitor.cpp:74)? The definitions are even different. If the difference is intentional, that's definitely worth a comment or at least renaming the functions to clarify their distinct behaviors. If the difference is not intentional, could this become `SymbolInfo::isAggregate`?

Reply:
Long term there shouldn't be this function at all and just a single type in the representation.  Right now the intermediate form can have both types (depending on how it was parsed) which needs to be addressed.

Reply:
In that case, a comment would definitely be advisable just to give future readers a clue that this is a stopgap and that in the future there should only be `RelationType::kAggregate`